README.md

```diff
@@ -36,22 +44,20 @@ To include extra testing dependencies, the syntax is as follows:
 
     pip install "jwst[test] @ git+https://github.com/spacetelescope/jwst"
 
-### Installing a DMS release ###
+### Installing a Data Management System (DMS) release ###
 
-We still package our releases to DMS via environment snapshots that specify the exact versions of all packages to be installed.
+We still package our releases to DMS via environment snapshots that specify the exact versions of all packages to be installed. This method should be more stable than the frist method because the latest versions of packages may cause problems.
 
 The latest release 0.15.0 may be installed by running the following commands in Linux or MacOS:
 
 Linux:
 
-    conda create -n jwstdp-0.15.0 --file https://ssb.stsci.edu/releases/jwstdp/0.15.0/conda_python_stable-deps.txt
-    conda activate jwstdp-0.15.0
+    conda install --file https://ssb.stsci.edu/releases/jwstdp/0.15.0/conda_python_stable-deps.txt
```

Comment:
I hope it remains clear to someone doing this that they should first create a specific conda environment in which they want to install this particular version of the package. Without creating and activating a particular environment, it's just going to get installed in whatever environment is currently active or (if no environment is active) in their "base" environment.

Reply:
We don't not want to make this change to the instructions.  We specifically recommend doing a `conda create` here so that the DMS environment is isolated.  We don't know what exists in an existing env and whether there will be conflicts with what is in the DMS build env.

Reply:
The problem is that the current instructions create a new conda env for the DMS installation but not for the "end user" installation or the developer installation. This lack of symmetry confused me. The revised instruction create the env at the top and then say to choose one of the three methods. I'll try adding something to make it clearer.

Reply:
My first draft put the env creation and activation at the start of each option. I removed it because it seemed redundant. I can go back to that, if people don't mind the redundancy. I do think the three options should be symmetrical.

Reply:
Symmetry is not a good enough argument for changing this.  We are giving here the *exact* same installation procedure that DMS uses to install it in their systems.  That is the purpose of having this method of installation available here - to debug DMS issues.  End users, INS testers can use this method if they like but it is not necessary.

Reply:
@jdavies-st The problem is that the instructions are not clear. It is not just me that thinks this is true. Others have commented on it too. I had to read it several times to figure out that the conda env created at the top needs to be used for the 1st and 3rd methods of installing but not for the second method. I've updated the new version to repeat the conda env creation and activation at the start of each of the three methods. 
So, should the DMS installation instructions say this is only for DMS?